### ISSUE TITLE
(feat) More crate alternatives!

### ISSUE THREAD

**Issue by peterheesterman** [peterheesterman/chit#34]
**Describe the solution you'd like**
We need to provide more crate alternatives suggestions! 

(Ideally people will make PRs suggesting sets of alternatives.)

**Comment by 8176135** [comment]
Maybe it can use the keyword thing on crates to automatically search for alternatives

**Comment by peterheesterman** [comment]
Yeah i am keen to see how this might look. I might make another command `chit keywords [some words]` or `chit keywords --crate=[something]` will do a search with all matching keywords to the crate `something`.  🤔 See if i find it useful myself at all.
